Marlin/Configuration.h

```diff
@@ -1658,7 +1658,13 @@
 #define Z_PROBE_ERROR_TOLERANCE     3 // (mm) Tolerance for early trigger (<= -probe.offset.z + ZPET)
 //#define Z_AFTER_PROBING           5 // (mm) Z position after probing is done
 
-#define Z_PROBE_LOW_POINT          -2 // (mm) Farthest distance below the trigger-point to go before stopping
+/*
+ * when probing for a mesh, homing must be performed ; this sets a reference triger point.
+ *
+ * If the bed is very uneven or if the probe fails to trigger, machine will stop moving
```

Comment:
Behavior varies if you have `G29_RETRY_AND_RECOVER` enabled, but it's more than just "stop moving" as Marlin will completely halt your machine.

Reply:
I will try with G29_RETRY_AND_RECOVER, without, it definitely keeps all motors blocked

Reply:
oops. I still need to update the comment in Configuration.h regarding the behaviour